quickstart-services.yml

```diff
@@ -51,12 +64,13 @@ services:
     depends_on:
       - kratos-migrate
     ports:
-      - "4433:4433" # public
-      - "4434:4434" # admin
+      - 4433
+      - 4434
     restart: unless-stopped
     environment:
       - DSN=mysql://root:${MYSQL_ROOT_PASSWORD}@tcp(${DATABASE_HOST}:3306)/kratos?max_conns=20&max_idle_conns=4
       - LOG_LEVEL=trace
+      - SERVE_PUBLIC_BASE_URL=http://localhost:3000/identity/ory/kratos/public
```

Comment:
Already configured from the kratos.yml

Reply:
Addressed in ff3914cf670151f476fbe9bdf9b747d578aec552. Removed.